.ci/Jenkinsfile

```diff
@@ -295,8 +295,6 @@ def generateStep(Map args = [:]){
             sh script: 'make build', label: 'Create releases'
           }
         }
-      } catch(e) {
-        error(e.toString())
```

Comment:
There is no need to catch the error so we reduce two step errors: the original one and the error step.